backend/api/models/users.js

```diff
@@ -6,11 +6,13 @@ const userSchema = new mongoose.Schema({
   email: {
     type: String,
     unique: true,
-    required: true
+    required: true,
+    maxlength: 36
```

Comment:
Tutaj dałbym 64 bo jak ktoś poda coś typu xdxdxd.student.uj.edu.pl to mu szybko zabraknie znaków

Reply:
Done